### ISSUE TITLE
Table of Contents?

### ISSUE THREAD

**Issue by TheHackerDev** [tomanistor/osprey#19]
Any chance you could add a table of contents to the theme? There's a guide here: https://gohugo.io/content-management/toc/.

Cheers,
Aaron ([insp3ctre](https://twitter.com/insp3ctre))

**Comment by tomanistor** [comment]
Yes I could add this in for an upcoming version. It would probably be displayed conditionally.

**Comment by tomanistor** [comment]
Unfortunately, Hugo's generated table of contents doesn't look like something I can add in at this time. This is because it adds a the table of contents into a `<nav>` tag. The theme already uses this tag for the fixed navbar at the top, so when Hugo generates the table of contents, it gets displayed inside of the navbar.
